pydantic/types.py

```diff
@@ -152,6 +152,35 @@ def conint(
 
     Returns:
         The wrapped integer type.
+
```

Comment:
We need to add a warning here about the fact that `conint` is discouraged.

I've already added the warning somewhere on this file, on another PR. If you can't find it, send me a message.